tools/autograd/templates/python_torch_functions.cpp

```diff
@@ -534,4 +497,91 @@ void initTorchFunctions(PyObject* module) {
   }
 }
 
+PyObject* handle_torch_function(PythonArgs &r, PyObject* args, PyObject* kwargs, PyTypeObject &torch_api) {
+  PyObject* torch_api_function =
+    PyObject_FastGetAttrString((PyObject*)&torch_api, const_cast<char*>(r.get_func_name().data()));
+  TORCH_INTERNAL_ASSERT(torch_api_function != NULL, "torch API function must exist");
```

Comment:
This assert is because you have a precondition that torch api function must exist; aka I want to make the same comment I made in the other location: please explicitly state your preconditions at the beginning of the function!